core/ClassLoader.php

```diff
@@ -1,13 +1,41 @@
 <?php
 
+//    以下エラーコード
+echo $miss_variablec
+
+// ここまで
+
 /**
  * ClassLoader.
  *
  * @author Katsuhiro Ogawa <fivestar@nequal.jp>
  */
 class ClassLoader
+class ClassLoader
```

Comment:
ERROR: Each class must be in a namespace of at least one level (a top-level vendor name)